graphql/Dockerfile

```diff
@@ -9,9 +9,12 @@ ENV PYTHONUNBUFFERED 1
 # Keep key consistent with value in uvicorn_settings.py
 ENV PORT 5000
 
-# Install python app requirements
+# Install psycopg2 requirements
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends build-essential libpq-dev
```

Comment:
Also check if we need to update the bamboo docker image so that it installs this package.
https://github.com/nasa/cumulus-orca/blob/develop/base_images/Dockerfile.bamboo

Reply:
psycopg2 is auto-installed for lambdas in AWS, so we don't need to worry about building/including it in this image.